modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/config/initialState.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {INITIAL_STATE} from 'dynamic-data-mapping-form-renderer/js/core/config/index.es';
+
+const DATA_ENGINE_INITIAL_STATE = {
+	...INITIAL_STATE,
+	editingLanguageId: themeDisplay.getDefaultLanguageId(),
+	rules: [],
```

Comment:
@diegonvs I think we can also move this to the core's initialState. What do you think? Since Forms does the same.

Reply:
Yes, makes sense. Moving!